### ISSUE TITLE
Shutdown session when depacketizer drops all packets

### ISSUE THREAD

**Issue by gavv** [roc-project/roc#109]
Currently it's possible that the packet stream is delayed a bit and depacketizer timestamp goes ahead it, and depacketizer starts dropping all packets that it gets from fec reader forever.

We should either fix it or at least detect it and shutdown the session, so it could be automatically restarted.

**Comment by gavv** [comment]
We probably can solve this by moving Watchdog from packet level to frame level (if we would mark silence frames).

**Comment by gavv** [comment]
Note: we should distinguish packet losses and packet delays. We want to restart the session when glitches are caused by delays, because restarting the session can fix the situation. However, we do not want to restart it when glitches are caused by packet loss, because it won't change anything and it's up to the user to decide whether to keep or shutdown the session.

**Comment by gavv** [comment]
Will be fixed by #115.

**Comment by gavv** [comment]
There are different reasons why depacketizer may start dropping all packets. Likely, all of them mean that we should restart the session.

One of them is described in #92. I've tested that the watchdog fixes it (see the issue), but it required disabling latency monitor and silence timeout.

However, there are other cases that can reproduce even with enabled latency monitor and silence timeout. I've tested that the watchdog correctly handles one of such cases.

What I did:

* hacked roc-sender so that it regularly inserts delays between packets;
* waited until depacketizer starts dropping all packets and checked the the watchdog detects it and shutdowns the session.

See details here: https://gist.github.com/gavv/a5727d5661f09b9b68da60ebe60c2ed8
